### ISSUE TITLE
Login Failed with the login Api

### ISSUE THREAD

**Issue by superLala** [mgp25/Instagram-API#1237]
when I use the api to login with my account ,I always get the message
{
  "message" : "Sorry, there was a problem with your request.",
  "status" : "fail",
  "error_type" : "sentry_block"
}
but I can login successfully with this account in the Instagram app.
Why it happened?
Does it mean Instagram is blocking the 3rd-party apps' login action?

**Comment by itsMoji** [comment]
https://github.com/mgp25/Instagram-API/wiki/FAQ#what-does-sentry_block-error-mean

> **What does "sentry_block" error mean?**

> This is Instagram's response when your account has been banned from the API for detected spam/bot behavior.

> The website and app will still work, but you will not be able to use that account via this API anymore. There is no way to get unblocked.

> Specifically, "sentry block" means that Instagram has blocked your account's ability to use this PHP API library (and all other reverse-engineered libraries like it), because of either:

    Spamming or otherwise abusing (such as mass-following)
    Or repeatedly and heavily misusing APIs, in a way humans would never call them.

> People using this library like a normal person don't get sentry blocked. Which means that you've had to > do something bad to get blocked by Instagram. There is no way to get unblocked.

> Stop using the API in whatever way that triggered this response.

**Comment by SteveJobzniak** [comment]
Thanks for helping with the reply, @itsMoji. :-)

What is going on in this ticket btw? It says it was opened on May 12, but itsMoji replied to it 26 minutes ago. And there are no opened/closed log notes that say it has been reopened. Wtf. I guess Github is bugged out?

**Comment by SteveJobzniak** [comment]
Yep this guy is a spammer. But I still wonder wtf happened here. xD

`superLala opened this issue on May 12`

**Comment by mgp25** [comment]
@SteveJobzniak drunk github is drunk

**Comment by SteveJobzniak** [comment]
@mgp25 Yep I even researched if there's some hidden `<form>` field to mark a ticket date that this guy used to fake the date, but I checked on the new ticket page with `var form = document.getElementById('new_issue'); var inp = form.getElementsByTagName('input'); console.log(inp);` and nothing like that exists. Github is drunk! xD
